src/controller/CHIPDeviceController.cpp

```diff
@@ -1724,13 +1726,49 @@ void DeviceCommissioner::OnDone()
     CommissioningStageComplete(return_err, report);
 }
 
+// TODO if we want to branch out the Commissioning Error to a specific chip error this will be
+// moved to the top and likely is better as a map lookup (which I would look for suggestions).
+CHIP_ERROR ConvertCommissioningErrorToChipError(GeneralCommissioning::CommissioningError commissioningError)
+{
+    switch (commissioningError)
+    {
+    case GeneralCommissioning::CommissioningError::kOk:
+        return CHIP_NO_ERROR;
+        break;
+    case GeneralCommissioning::CommissioningError::kValueOutsideRange:
+        return CHIP_ERROR_INVALID_ARGUMENT; //  TODO figure out more appropriate error.
+        break;
+    case GeneralCommissioning::CommissioningError::kInvalidAuthentication:
+        return CHIP_ERROR_INVALID_ARGUMENT; //  TODO figure out more appropriate error.
+        break;
+    case GeneralCommissioning::CommissioningError::kNoFailSafe:
+        return CHIP_ERROR_INVALID_ARGUMENT; //  TODO figure out more appropriate error.
+        break;
+    case GeneralCommissioning::CommissioningError::kBusyWithOtherAdmin:
+        return CHIP_ERROR_INVALID_ARGUMENT; //  TODO figure out more appropriate error.
```

Comment:
suggest CHIP_ERROR_BUSY here.

Reply:
Removed this mapping, no longer applicable